features/search-replace.feature

```diff
@@ -15,6 +15,16 @@ Feature: Do global search/replace
       wp_posts
       """
 
+    When I run `wp search-replace foo bar --skip-tables=wp_post\*`
+    Then STDOUT should not contain:
```

Comment:
Same here, a positive match should be added to make sure it works as expected, not just kills output.